api/containerserver/hijack.go

```diff
@@ -192,6 +195,14 @@ func (s *Server) hijack(hLog lager.Logger, conn *websocket.Conn, request hijackR
 				}
 			} else {
 				_, _ = stdinW.Write(input.Stdin)
+				if s.interceptIdleTimeout != 0 {
+					idle.Reset(s.interceptIdleTimeout)
```

Comment:
This can probably just be right after the `case` line; it might as well apply to all inputs.